wayland-protocols/CHANGELOG.md

```diff
@@ -5,7 +5,7 @@
 ### Breaking changes
 
 - Bump bitflags to 2.0
-- Updated wayland-backend to 0.2
+- Updated wayland-backend to 0.3
```

Comment:
This looks arbitrary and unrelated to this PR, should be submitted separately.

Reply:
Well, this PR introduces breaking changes that cascade to all other crates, so it seems appropriate to add the associated changelog entries in the same PR.

Reply:
But you don't actually bump the version here, so it should be done in a separate commit.

Reply:
ok